tests/spicy/types/vector/infinite-loop-for-each.spicy

```diff
@@ -0,0 +1,17 @@
+# @TEST-EXEC: spicyc -dj -o x.hlto %INPUT
+# @TEST-EXEC-FAIL: printf '\x00\x02\x03' | spicy-driver x.hlto >>output 2>&1
+# @TEST-EXEC:      printf '\x02\x02\x03' | spicy-driver x.hlto >>output 2>&1
```

Comment:
This seems to call an incorrect `printf`. Calling `${SCRIPTS}/printf` instead makes this pass for me.

We seem to in a number of places call naked `printf` instead of our wrapper. I'll check whether adjusting these would also allow to bump the spicy-analyzers submodule. Alternatively, we might be able to add `${SCRIPTS}` to `PATH` so this misuse becomes less likely in the future.